pkgs/by-name/xt/xterm/package.nix

```
@@ -12,16 +12,20 @@
   pkgsCross,
   gitUpdater,
   enableDecLocator ? true,
+  gnused,
+  gawk,
+  glibc,
+  bashNonInteractive,
```

or should I turn this into
```suggestion
  bashNonInteractive,
  bash ? bashNonInteractive,
```
and use plain `bash` everywhere else for a bit more elegant overridability?